packages/nextjs-mf/src/plugins/container/InvertedContainerPlugin.ts

```diff
@@ -166,7 +166,9 @@ class InvertedContainerPlugin {
               //@ts-ignore
               renderContext?._name ||
               !renderContext?.debugId ||
-              !compilation.chunkGraph.isEntryModule(renderContext)
+              !compilation.chunkGraph.isEntryModule(renderContext) ||
+              //@ts-ignore
+              renderContext?.rawRequest?.includes('pages/api')
```

Comment:
will work for now, but need to find better ways to track appliciable chunks or add the same container inversion system to the api runtime as well